integration-tests/src/test/scala/za/co/absa/spline/CobrixWriteSpec.scala

```diff
@@ -0,0 +1,108 @@
+      |                      15  ACCOUNT-NUMBER     PIC X(24).
+      |                      15  ACCOUNT-TYPE-N     PIC 9(5) COMP-3.
+      |                      15  ACCOUNT-TYPE-X     REDEFINES
+      |                           ACCOUNT-TYPE-N  PIC X(3).
+      |
+      |""".stripMargin
+
+  val bytes: Array[Byte]  = Array[Byte](
+    0x00.toByte, 0x06.toByte, 0xC5.toByte, 0xE7.toByte, 0xC1.toByte, 0xD4.toByte, 0xD7.toByte, 0xD3.toByte,
+    0xC5.toByte, 0xF4.toByte, 0x40.toByte, 0x40.toByte, 0x00.toByte, 0x00.toByte, 0x0F.toByte, 0x40.toByte,
+    0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte,
+    0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte,
+    0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte, 0x40.toByte,
+    0x00.toByte, 0x3F.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF2.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF4.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF1.toByte, 0xF2.toByte, 0x00.toByte, 0x00.toByte, 0x0F.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF3.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF4.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF1.toByte, 0xF0.toByte, 0xF2.toByte, 0x00.toByte, 0x00.toByte, 0x1F.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0xF5.toByte, 0xF0.toByte, 0xF0.toByte, 0xF6.toByte, 0xF0.toByte, 0xF0.toByte, 0xF1.toByte, 0xF2.toByte,
+    0xF0.toByte, 0xF0.toByte, 0xF3.toByte, 0xF0.toByte, 0xF1.toByte, 0xF0.toByte, 0xF0.toByte, 0xF0.toByte,
+    0x00.toByte, 0x00.toByte, 0x2F.toByte
+  )
+
+  val bos = new FileOutputStream(cobrixPath)
+  bos.write(bytes)
+  bos.close()
+// taggedAs ignoreIf(ver"$SPARK_VERSION" < ver"2.4.2")
+  it should "support Cobrix as a source" in
+    withNewSparkSession(spark => {
+      withLineageTracking(spark)(lineageCaptor => {
+
+        val (plan1, _) = lineageCaptor.lineageOf(spark
+          .read
+          .format("cobol")
+          .option("copybook_contents", copybook)
+          .option("is_record_sequence", "true")
+          .option("schema_retention_policy", "collapse_root")
+          .option("segment_id_" +
+          "level0", "R")
+          .option("segment_id_prefix", "ID")
+          .load(cobrixPath).write.mode(Overwrite).saveAsTable("somewhere")
+        )
+        plan1.operations.write.append shouldBe false
+        plan1.operations.reads.get.head.inputSources.head shouldBe cobrixPath
+        plan1.operations.reads.get.head.extra.get("sourceType") shouldBe Some("cobrix")
+
+
+
+      })
+    })
+}
```

Comment:
Please add a line break at the end of the file, it's a good practice https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#729795